input/cql/DetermineNextDueDateEvents.cql

```diff
@@ -0,0 +1,23 @@
+library DetermineNextDueDateEvents version '1.0.0'
+
+using FHIR version '4.0.1'
+
+include FHIRHelpers version '4.0.1' called FHIRHelpers
+include DataElements version '1.0.0' called DataElements
+
+context Patient
+
+define "eve-lastTestColonoscopyAvg":
```

Comment:
I'm assuming this event name came from the L2? If so, it's fine but was just wondering why this name has 'Avg' and the other modalities don't

Reply:
Yeah that comes directly from the events table in the L2: https://gitlab.mitre.org/cancer-cds/colorectal-cancer/cdc-colorectal-cancer-screening-cds-l2/-/blob/main/input/data/events.csv?ref_type=heads&plain=1#L49. I wasn't sure why it was there either. In https://github.com/CancerScreeningCDS/crcsm-cds/commit/c26a86c7b85fd8472cf33bc52e78b02a774ee83f, I went ahead and did a find and replace to remove the "Avg" part.